### DISCUSSION TITLE
Audio link

### DISCUSSION THREAD

**Discussion by ZB2ham** [N0BOY/FT8CN#53] [Show and tell]
I am sure many have started using FT8CN by holding a smartphone to the radio's speaker, and the radio's microphone to the 'phone's speaker. It is amazing! Then of course you wish to go hands-free and link the radio to the 'phone or tablet with an appropriate cable. I did that and could NOT get the thing to work - because it turns out, Android 12 no longer presents audio at the 'phone's USB connector. 

My solution, which it turns out, has been done before, is to make an audio patch cable. Details can be found at
https://www.youtube.com/watch?v=hNZCmg2lXpQ&ab_channel=AmateurRadioVK3YE

So I built this - very simple as you can see - and it works very well indeed! I tried it on several 'phones and no problem. There are advantages....

1. Works with all 'phones and tablets without the need for software installation or selecting rigs or settings in the app
2. As the radio's and the smartphone's speaker connections are muted by insertion of the relevant plug AND the radio mic is no longer used, no extraneous noises will interfere with transmission or reception.

Disadvantage - the radio and smartphone/tablet no longer "talk" to each other down the data cable, a change of operating frequency on one device is NOT reflected in the other. You need to change the operating frequency on both - in FT8CN for the purposes of the log only, of course.

NOTE - I DO NOT GUARANTEE OR ENDORSE THIS AS A SOLUTION, ONLY STATING THAT IT WORKED FOR ME. BE AWARE TOO HIGH A VOLUME SETTING AT EITHER END OF THE AUDIO INTERFACE MAY CAUSE DAMAGE TO THE PHONE OT TABLET OR RADIO. ALSO, I HAVE NO CONTROL OVER YOUR CONSTRUCTION SKILLS OR CHOICE OF COMPONENTS
![pic1](https://github.com/N0BOY/FT8CN/assets/132378578/8d55e39a-78c0-4f6a-8f98-57f3772bf4f4)
![Pic2](https://github.com/N0BOY/FT8CN/assets/132378578/b89dc02d-449c-4559-9f6c-3fd4d8acda5d)
!!

**Comment by Wirezfree** [comment]
Hi,

Used have one of these SmartPhone Interfaces to Run Droid Radio Apps
Wish I'd kept it :(

PDF.. Description / Schematic
https://www.wolphi.com/wolphilink/wolphilinkmanual.pdf

Best 73 - Dave - G8LIY

**Comment by dinosbg** [comment]
Hello, i use th usb Vox 1.00 : https://antrak.org.tr/blog/usb-vox-1-00-usb-sound-card-vox-combo/  with a USB C to USB A adapter for my Phone (VIVO Y36). The Phone recongnized the soundcard und Interface generate PTT signal.
Work very good with my China HF QRP TRX.
![IMG_20241016_085136](https://github.com/user-attachments/assets/75efd053-30d2-42df-94c5-54795554e73a)
![IMG_20241016_085044](https://github.com/user-attachments/assets/4269a688-d0f6-42e5-b3d1-8250467c5fd4)
.
